server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -548,6 +553,7 @@ export class ScribeLambda implements IPartitionLambda {
 			minimumSequenceNumber: this.minSequenceNumber,
 			protocolState,
 			sequenceNumber: this.sequenceNumber,
+			validParentSummaries: this.validParentSummaries,
```

Comment:
This is backwards compatible. I left the type for `validParentSummaries` as `string[] | undefined` so that I would be forced to make sure I handle all cases where `validParentSummaries` is undefined (e.g. for existing checkpoints).